### DISCUSSION TITLE
Add config option for how EssX saves worlds

### DISCUSSION THREAD

**Discussion by Chris6ix** [EssentialsX/Essentials#4311] [Ideas and Feature Suggestions] [answered]
Hey!
I wanted to ask if you guys could please add a small config option in EssentialsX 2.19.x that lets you choose how EssX saves worlds in the userdata files.

This config option would have 2 options: `UUID` and `Name`
The default would be `UUID` to match the current behavior of EssX 2.19.x but changing this option to `Name` would make it so that EssX uses the "old" method from EssX 2.18.2 to save world names for `logoutlocation:world` and `homes:"HomeName":world`.
Please see the chat in #4310 for more information.

**Comment by pop4959** [comment, marked as answer]
The fact that the userdata files use world names at all is extremely dated, and the fact that it has just been updated now is surprising to say in the least (although better late than never). This should have been changed already years ago when the playerdata files were initially converted from using player names to player UUIDs (player.yml ->  uuid.yml). Numerous issues are also fixed by saving by UUID such as homes remaining valid even when the world is swapped out for a completely different one with the same name. It makes zero sense to store by world names anymore and a configuration option will not be added.
